<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100,100,100);
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>507</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_9">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>19</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Head Detection Calibration</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="close_btn">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	border: none;
	background-color: rgb(100,100,100);
}

QPushButton:hover {
	background-color: rgb(255, 38, 0);
}

QPushButton:pressed {
	background-color: rgb(255, 0, 0);
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resources.qrc">
                    <normaloff>:/icons/icons/icons8_multiply.svg</normaloff>:/icons/icons/icons8_multiply.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QFrame" name="frame_3">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>9</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Original Image</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_getpicture_2">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>111</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>111</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Live</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_getpicture">
                     <property name="minimumSize">
                      <size>
                       <width>111</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>111</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Get Picture</string>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>images/play.png</normaloff>images/play.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label_head_calibration">
                  <property name="maximumSize">
                   <size>
                    <width>1000</width>
                    <height>500</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Box</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>images/dorsa2.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_9">
                     <property name="minimumSize">
                      <size>
                       <width>81</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Real Image : </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_8">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <item>
                       <widget class="QLabel" name="label_2">
                        <property name="minimumSize">
                         <size>
                          <width>45</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Start :  </string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_3">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>X :</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_first_x"/>
                      </item>
                      <item>
                       <widget class="Line" name="line_2">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Y :</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_first_y"/>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_10">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="minimumSize">
                         <size>
                          <width>45</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>End  :  </string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>X :</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_end_x"/>
                      </item>
                      <item>
                       <widget class="Line" name="line_4">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_8">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Y :</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_end_y"/>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="show_messages">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_save">
                     <property name="text">
                      <string>  Save </string>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>images/1206141891556277128-128.png</normaloff>images/1206141891556277128-128.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
